components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/MenuButton.kt

```diff
@@ -113,4 +120,14 @@ internal class MenuButton @JvmOverloads constructor(
     fun setColorFilter(@ColorInt color: Int) {
         menuIcon.setColorFilter(color)
     }
+
+    private fun hasClickThrottleDurationPassed(lastClickTime: Long): Boolean {
+        return now() - lastClickTime >= MENU_CLICK_THROTTLE_DURATION
+    }
+
+    private fun now() = SystemClock.elapsedRealtime()
+
+    private companion object {
+        const val MENU_CLICK_THROTTLE_DURATION = 500
```

Comment:
The title says 300 and here its 500. :)

Reply:
Thanks.
I initially went with 300ms but when testing more I saw that maybe it isn't enough.
On the other side 500 should cause any issues.